compiler/daml-extension/BUILD.bazel

```diff
@@ -73,6 +73,10 @@ genrule(
     tools = ["//bazel_tools/sh:mktgz"],
 )
 
+# This rule is not reproducible. `vsce package` generates a `.vsix` file which
+# is just a zip archive. The order of entries in that archive is
+# non-deterministic and the individual entries contain non-reproducible
+# timestamps.
```

Comment:
Could we unzip and rezip to fix this?

Reply:
Yes, I was hoping to find a way to do this with vsce directly, but failing that this should do it.

Reply:
Not much to be gained trying to fix this if we cannot fix Navigator. In the end, the thing we really want to be reproducible is the SDK tarball.